src/ng/filter/orderBy.js

```diff
@@ -89,9 +92,7 @@
  *
  * **Note:** If you notice numbers not being sorted as expected, make sure they are actually being
  *           saved as numbers and not strings.
- * **Note:** For the purpose of sorting, `null` values are treated as the string `'null'` (i.e.
- *           `type: 'string'`, `value: 'null'`). This may cause unexpected sort order relative to
- *           other values.
+ * **Note:** `null` and `undefined` values are always treated bigger than any other value, where `undefined` is treated bigger than `null`.
```

Comment:
I would change it to something like:

> For the purpose of sorting, `null` and `undefined` are considered "greater than" any other value (with `undefined` "greater than" `null`). This effectively means that `null` and `undefined` values end up at the end of a list sorted in ascending order.

Reply:
I would also add another note about `null` values using `'null'` as their type (to more easily distinguish them from objects).

Reply:
What about: https://github.com/angular/angular.js/pull/16376/files#diff-131b4811903f9fb834e9994a4386dc05R43 ?

Reply:
I think that's fine. I would just add another note here.

Reply:
Done.